tutordiscovery/plugin.py

```diff
@@ -31,6 +31,7 @@
         "OAUTH2_KEY_SSO": "discovery-sso",
         "OAUTH2_KEY_SSO_DEV": "discovery-sso-dev",
         "CACHE_REDIS_DB": "{{ OPENEDX_CACHE_REDIS_DB }}",
+        "ATLAS_PULL": False,
```

Comment:
@OmarIthawi , shouldn't this be `OPENEDX_ATLAS_PULL` ?

Reply:
@shadinaif the prefix is automatically added via Tutor. So in the `config.yaml` file it will be `DISCOVERY_ATLAS_PULL: true`.

Reply:
Thank you!